app/src/examples/DynamicStylesExample.tsx

```diff
@@ -0,0 +1,186 @@
+import { Text, StyleSheet, View, Button } from 'react-native';
+
+import React from 'react';
+
+import Animated, {
+  useAnimatedStyle,
+  useSharedValue,
+  withTiming,
+} from 'react-native-reanimated';
+
+export default function DynamicStylesExample() {
+  const [extraStyle, setState] = React.useState(false);
+  const [rerender, setRerender] = React.useState(false);
```

Comment:
Let's separate React state from shared state by adding a newline here.
```suggestion
  const [rerender, setRerender] = React.useState(false);

```